pkgs/development/libraries/qt-5/5.8/qtbase/default.nix

```diff
@@ -15,7 +17,8 @@
   cups ? null, mysql ? null, postgresql ? null,
 
   # options
-  mesaSupported, mesa,
+  mesaSupported ? (!stdenv.isDarwin),
```

Comment:
`mesaSupported` and `mesaPlatforms` are just bad names.  On Darwin we do *not* support mesa itself, but there's the libGL* from the OS that we do support, provided at the same `mesa_noglu` attribute.